drafts/use-cases/5.1.creating-new-event-indirectly.md

```diff
@@ -0,0 +1,128 @@
+## Creating new event indirectly
+
+### Story
+
+As an API publisher
+I want allow consumers to construct event's identifier
+So that they can control the .
+
+As an API consumer
+I want to be able to create new resources of a given type
+So I can produce content.
+
+### Usage
+
+``` js
+var event = {
+    "@context": "/api/context.jsonld",
+    "@type": "schema:Event",
+    "eventName": "My brand new event",
+    "eventDescription": "Hope it will work",
+    "startDate": "2017-04-19",
+    "endDate": "2017-04-19"
+};
+var templateVariables = {
+    "slug": [
+        "meeting",
+        "with-will"
+    ]
+};
+
+var client = new HydraClient();
+var operation = client.get("http://example.com/events")
+    .getOperationOfType('http://schema.org/CreateAction', /* recursive */ true)
+    .filter(operation => operation.expects === 'http://schema.org/Event');
+    
+client.invoke(operation, event, templateVariables);
+```
+
+### Details
+
+To create an `Event` using `PUT`, the collection resource has to be linked to the actual target of the operation.
+This linked object can be a concrete resource or an `IriTemplate` as presented in the above snippet. In this case 
+the representation of the collection resource is not unlike that seen in the [Searching events use case](7.searching-events.md)
+
+```http
+GET /api/events
+```
+
+```http
+HTTP 200 OK
+```
+
+``` json
+{
+    "@context": "/api/context.jsonld",
+    "@id": "/api/events",
+    "@type": "hydra:Collection",
+    "manages": {
```

Comment:
I somehow have some doubts about this `manages` construct. What does it say? Are all resources of type `schema:Event` members of this collection? Or should I add each member explicitly? I believe both cases are pretty legal - server could expose on-the-fly views acting as collections just by using some of the resource's attributes (i.e. collection of wines where each resource is of type `foo:Wine`), but it also could expose explicit collections (i.e. my likes that I've added manually).